SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceEvent.h

```diff
@@ -2,20 +2,49 @@
 
 #pragma once
 
-namespace SpatialGDK
+#include "CoreMinimal.h"
+
+#include "SpatialTraceEvent.generated.h"
+
+USTRUCT(Blueprintable)
+struct FTraceData
 {
+	GENERATED_BODY()
+
+	FTraceData(){};
+	explicit FTraceData(FString InKey, FString InValue)
+		: Key(MoveTemp(InKey))
+		, Value(MoveTemp(InValue))
```

Comment:
Can we use const FString& InKey, const FString& InValue instead? to make MoveTemp unnecessary.

Reply:
I think pass-by-value is a good call here - there is an option to MoveTemp into InKey and InValue which would incur no copy/allocation costs - which happens here https://github.com/spatialos/UnrealGDK/pull/2625/files#diff-82582710edb0f901de5f00d3007a7cfa7eeb7d031425675de5ba30dc8fa72d52R40